bfvmm/include/hve/arch/intel_x64/vmexit/control_register.h

```diff
@@ -136,7 +84,32 @@ class EXPORT_HVE control_register_handler
 
 public:
 
-    /// Add Write CR0 Handler
+    /// Add wrcr0 Handler
+    ///
+    /// Adds a VM exit handler. When a VM exit occurs, the registered
```

Comment:
Not a fan of the copy-pasted comments. Can consolidate this info to a single place, then add exit-specific comments as needed here (and elsewhere)?

Reply:
I'd like to do both. It is really important that the details of each of these are well documented. We should also provide this same documentation in our examples. More documentation is a good thing IMO, this way no matter what file you land on, you will get the needed information to properly implement a handler/emulator

Reply:
I'm all for documenting in every file, but if something is copy-pasted more than once within that file, I think it should just be put once at the top or something